### PR TITLE
Working calico ipip configuration

### PR DESCRIPTION
**Pull request by ktsakalozos**
IPinIP option in the initConfiguration of nodes

### CODE REVIEW

File: controllers/microk8sconfig_controller.go [hunk 1]
```diff
@@ -396,6 +400,11 @@ func (r *MicroK8sConfigReconciler) handleJoiningControlPlaneNode(ctx context.Con
 		controlPlaneInput.HTTPSProxy = microk8sConfig.Spec.InitConfiguration.HTTPSProxy
 		controlPlaneInput.HTTPProxy = microk8sConfig.Spec.InitConfiguration.HTTPProxy
 		controlPlaneInput.NoProxy = microk8sConfig.Spec.InitConfiguration.NoProxy
+
+		if microk8sConfig.Spec.InitConfiguration.IPinIP {
+			controlPlaneInput.IPinIP = true
+		}
```
Comment (neoaggelos) [inline controllers/microk8sconfig_controller.go:406]:
```suggestion
		controlPlaneInput.IPinIP = microk8sConfig.Spec.InitConfiguration.IPinIP
```
any reason why not this?

**Review by sachinkumarsingh092 — APPROVED** [review]
LGTM.
But we certainly need some tests to be created for the controllers soon so as to include tests in future PRs. WDYT?

File: controllers/microk8sconfig_controller.go (hunk 1 above)
Comment (ktsakalozos) [reply controllers/microk8sconfig_controller.go]: Done

**Review by neoaggelos — APPROVED** [review]
LGTM

**Pull request merged**